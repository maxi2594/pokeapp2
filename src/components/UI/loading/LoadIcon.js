import React from 'react'
import './loadIcon.css'

export const LoadIcon = () => {

    return (
        <div className="lds-dual-ring"></div>
    )
}
