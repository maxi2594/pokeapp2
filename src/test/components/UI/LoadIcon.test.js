import { shallow } from 'enzyme'
import React from 'react'
import { LoadIcon } from '../../../components/UI/loading/LoadIcon'

describe('tests in <LoadIcon/>', () => {
    
    const wrapper = shallow(<LoadIcon/>)

    test('should be shown properly', () => {
        
        expect( wrapper ).toMatchSnapshot();

        expect( wrapper.find('div').props().className ).toBe('lds-dual-ring')

    })
    
})
