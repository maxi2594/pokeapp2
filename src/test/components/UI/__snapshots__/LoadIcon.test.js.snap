// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests in <LoadIcon/> should be shown properly 1`] = `
<div
  className="lds-dual-ring"
/>
`;
